2333/2/versus.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'libs/options_parser.rb'
+require_relative 'libs/handler.rb'
+require_relative 'libs/top_bad_words_analyzer.rb'
+require_relative 'libs/top_words_analyzer.rb'
+
+parameters = ParametersParser.new.options
+rappers = Handler.create_rappers_array
+DEFAULT_AMOUNT = 30
+
+if parameters[:top_bad_words]
+  analyzer = TopBadWordsAnalyzer.new(rappers, parameters[:top_bad_words])
+  analyzer.top_bad_words
+elsif parameters[:name]
+  parameters[:top_words] ||= DEFAULT_AMOUNT
```

Comment:
This is not what I meant by default arguments. Use method default arguments: https://learn.co/lessons/methods-default-arguments.